app/__init__.py

```diff
@@ -32,7 +32,10 @@ def index():
     db.init_app(app)
     migrate.init_app(app, db)
     jwt.init_app(app)
-    from app.auth import auth_blueprint
 
-    app.register_blueprint(auth_blueprint)
+    # Register blueprints
+    from app import auth, user_task
```

Comment:
This looks like it might cause a circular dependecy!
As from `app` we import `auth` but running `auth` requires that `db` is imported from `app`.